examples/WORKSPACE

```diff
@@ -107,5 +107,8 @@ stack_snapshot(
         "text-show",
     ],
     snapshot = "lts-14.0",
+    # Pin Stackage dependencies with `bazel run @stackage-unpinned//:pin`.
```

Comment:
```suggestion
    # This example uses an unpinned version of Stackage, meaning that stack is invoked on every build.
    # To switch to pinned stackage dependencies, run `bazel run @stackage-unpinned//:pin` and
    # uncomment the following line. 
```

Reply:
Thanks! I've rephrased it slightly. The phrasing "version of Stackage" is a bit confusing as it may be interpreted as "Stackage snapshot version". The Stackage snapshot is already pinned with the `snapshot` and `local_snapshot` attribute. `stack_snapshot_json` additionally pins the package metadata that `stack` already knows in a way that's accessible to Bazel so that `stack` no longer needs to be invoked.